lib/use_packwerk/private.rb

```diff
@@ -197,19 +197,7 @@ def self.make_public!(paths_relative_to_root:, per_file_processors:)
 
 
           file_move_operations = file_paths.map do |path|
-            parts = path.to_s.split('/')
-            first_part_of_path = T.must(parts[0])
-
-            if Pathname.new(first_part_of_path).dirname.join(ParsePackwerk::PACKAGE_YML_NAME).exist?
-              package_location = Pathname.new('.')
-            elsif PERMITTED_PACK_LOCATIONS.include?(first_part_of_path)
-              package_location = Pathname.new(first_part_of_path).join(T.must(parts[1]))
-            else
-              raise StandardError.new('Can only make files in the monolith or in existing packs public')
-            end
-
-            package = ParsePackwerk::Package.from(package_location.join(ParsePackwerk::PACKAGE_YML_NAME))
-
+            package = T.must(ParsePackwerk.package_from_path(path))
```

Comment:
The above was just a way of finding what package the file belonged to. Now that `ParsePackwerk` implements this, this could all be removed.